docs_src/design/adr/system-management/0006-Metrics-Collection.md

```diff
@@ -208,27 +201,38 @@ PublishTopicPrefix = "edgex/events/core" # standard and existing core or device
   SkipCertVerify = "false" # Only used if Cert/Key file or Cert/Key PEMblock are specified
 ```
 
-Additional configuration must be provided (in the service configuration.toml) to 
+Additional configuration must be provided in each service to provide metrics / telemetry specific configuration.  This area of the configuration will likely be different for each type of service.
+
+**Additional metrics collection configuration** to be provided include: 
+
 - Trigger the collection of telemetry from the metrics cache and sending it into the appointed message bus.
-- Metrics will be published to an /edgex/telemetry/[service-name]/[metric-name] topic where the service name will be added per service and the metric name per metric (allowing subscribers to filter by service or metric name)
+- Define which metrics are available and which are turned `off` and `on`.  All are false by default.  The list of metrics can and likely will be different per service.  The keys in this list are the metric name.  True and false are used for `on` and `off` values.
+- Specify the metrics topic prefix where metrics data will be published to (ex:  providing the prefix /edgex/telemetry/topic name where the service and metric name `[service-name]/[metric-name]` will be appended per metric (allowing subscribers to filter by service or metric name)
+
+These metrics configuration options will be defined in the `Writable` area of `configuration.toml` so as to allow for dynamic changes to the configuration (when using Consul).  Specifically, the `[Writable].[Writable.Telemetry]` area will dictate metrics collection configuration like this:
 
 ``` yaml
-[[Telemetry]]
-Interval = "30s"
-TelemetryPublishTopicPrefix  = 'edgex/telemetry' # /<service-name>/<metric-name> will be added to this Publish Topic prefix
+[[Writable]]
+    [[Writable.Telemetry]]
+    Interval = "30s"
+    TelemetryPublishTopicPrefix  = "edgex/telemetry" # /<service-name>/<metric-name> will be added to this Publish Topic prefix
+    Metrics = "{\"service-up\":\"false\", \"api-requests\":\"false\"}"  #all metrics will be off (or false) by default configuration
 ```
 
+!!! Info
+    It was discussed that in future EdgeX releases, services may want separate message bus connections.  For example one for sensor data and one for metrics telemetry data.  This would allow the QoS and other settings of the message bus connection to be different. This would allow sensor data collection, for example, to be messaged with a higher QoS than that of metrics.  For the initial release of this feature, the service will use the same connection (and therefore configuration) for metrics telemetry as well as sensor data.  
+

```

Comment:
We could also enhance go-mod-messaging to allow setting QOS on a per topic basis to avoid multiple connections.

Reply:
Added additional alternate suggestion to the Info comment.